### PR TITLE
Release cockroachdb 1.0.1 (automated commit)



### PR DESCRIPTION
**Pull request by OhRobin**
Release cockroachdb 1.0.1 (automated commit)

Description:
add cockroachdb framework

Changes since revision -1:
4 files added: [config.json, marathon.json.mustache, package.json, resource.json]
0 files removed: []
0 files changed:

```
```

### CODE REVIEW

**Comment by OhRobin** [comment]
cc: @springle @lakecarrot @a-robinson

File: repo/packages/C/cockroachdb/0/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "description": "CockroachDB on DC/OS",
```
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: please add a better description of what this package is about.

File: repo/packages/C/cockroachdb/0/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "framework": true,
+  "maintainer": "alex@cockroachlabs.com",
```
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: should be a cockroachlabs support/team email.

File: repo/packages/C/cockroachdb/0/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "postInstallNotes": "CockroachDB is being installed!\n\n\tDocumentation: https://www.cockroachlabs.com/docs\n\tIssues: contact https://www.cockroachlabs.com/docs/support-resources.html\n\tRelease Notes:https://www.cockroachlabs.com/docs/v1.0.2.html",
+  "postUninstallNotes": "CockroachDB has been uninstalled.",
+  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!",
+  "selected": false,
+  "tags": [
+    "cockroachdb",
+    "sql",
+    "database"
+  ],
+  "version": "1.0.1"
```
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: `version` should be <integration-version>-<base-tech-version>. So if cockroachdb version is 1.2.3 then, -> 1.0.1-1.2.3

File: repo/packages/C/cockroachdb/0/package.json (hunk 1 above)
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: add a link to the service guide.

File: repo/packages/C/cockroachdb/0/package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "description": "CockroachDB on DC/OS",
+  "framework": true,
+  "maintainer": "alex@cockroachlabs.com",
+  "minDcosReleaseVersion": "1.10",
+  "name": "cockroachdb",
+  "packagingVersion": "3.0",
```
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: change `packagingVersion` to 4.0

File: repo/packages/C/cockroachdb/0/package.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (ryadav88) [inline repo/packages/C/cockroachdb/0/package.json]: How would customers get licenses for CockroachDB Enterprise ?

File: repo/packages/C/cockroachdb/0/package.json (hunk 1 above)
Comment (OhRobin) [inline repo/packages/C/cockroachdb/0/package.json]: Added a better description. the service guide is https://github.com/dcos/examples/pull/195 and still haven't been merged yet

File: repo/packages/C/cockroachdb/0/package.json (hunk 2 above)
Comment (OhRobin) [inline repo/packages/C/cockroachdb/0/package.json]: cannot find support team email on website. 

cc: @a-robinson

File: repo/packages/C/cockroachdb/0/package.json (hunk 4 above)
Comment (OhRobin) [inline repo/packages/C/cockroachdb/0/package.json]: updated

File: repo/packages/C/cockroachdb/0/package.json (hunk 5 above)
Comment (OhRobin) [inline repo/packages/C/cockroachdb/0/package.json]: From the website, "The source code for CockroachDB Enterprise is packaged in the same binary as CockroachDB Core" and that they should contact https://www.cockroachlabs.com/pricing/sales/ 

Anything else to add @a-robinson ?

File: repo/packages/C/cockroachdb/0/package.json (hunk 3 above)
Comment (OhRobin) [inline repo/packages/C/cockroachdb/0/package.json]: updated

File: repo/packages/C/cockroachdb/0/package.json (hunk 5 above)
Comment (a-robinson) [inline repo/packages/C/cockroachdb/0/package.json]: I think @ryadav88 may have been requesting that that info goes somewhere in this file, but that info is correct.

File: repo/packages/C/cockroachdb/0/package.json (hunk 2 above)
Comment (a-robinson) [inline repo/packages/C/cockroachdb/0/package.json]: support@cockroachlabs.com will work

File: repo/packages/C/cockroachdb/0/package.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "CockroachDB on DC/OS",
+  "framework": true,
+  "maintainer": "alex@cockroachlabs.com",
+  "minDcosReleaseVersion": "1.10",
```
Comment (a-robinson) [inline repo/packages/C/cockroachdb/0/package.json]: I thought this worked with 1.9 as well? 1.10 hasn't even been released yet.

**Comment by OhRobin** [comment]
closed because I was working with the automated release script 
https://github.com/mesosphere/dcos-commons/tree/master/tools#release_builderpy. Reopening and will attempt to make revisions in the same automated branch

**Comment by OhRobin** [comment]
Please refer to https://github.com/mesosphere/universe/pull/1303

**Pull request closed without merging**